README.md

```diff
@@ -44,6 +44,33 @@ The Oracle WebLogic Server Deploy Tooling is designed to support a wide range of
 - The attribute data types and how to get/set their values (which isn't as easy as it might sound)
 - The differences between WLST online and WLST offline for working with folders and attributes
 
+### Supported WebLogic Server Releases
+The chart below specifies the supported WebLogic Server release, along with the JDK version(s) that must be used to run the WDT tool. You must set the JAVA_HOME environment variable to specify a version of a JDK different than the system default version. 
+
+ To create a domain with the proper JDK (particularly if the JAVA_HOME is different than the one which will be used by the target domain), set the Domain JavaHome attribute in the domain model.
+ 
+ Note that the WDT Encryption tool used to encrypt / decrypt clear passwords in the model and variable file requires WDT to run with a minimum JDK version of 1.8. 
```

Comment:
encrypt / decrypt clear passwords -> encrypt and decrypt clear text passwords
and variable file requires -> and variable file, requires